LICENSE

```diff
@@ -0,0 +1,21 @@
+MIT License
+
+Copyright (c) 2023 anmode
+
```

Comment:
This is wrong! please check the other licenses...and one thing to keep in mind this is a startup project. Thanks